src/main/java/com/github/lunatrius/schematica/client/util/BlockList.java

```diff
@@ -47,7 +47,16 @@
             final boolean isPlaced = BlockStateHelper.areBlockStatesEqual(blockState, mcBlockState);
 
             ItemStack stack = null;
-
+            
+            if (block.getIdFromBlock(block) == 43 || block.getIdFromBlock(block) == 125 || block.getIdFromBlock(block) == 181 || block.getIdFromBlock(block) == 204) {
```

Comment:
This is definitely a no. I am against any special cases where possible, but using the block IDs directly is a definite no go.

I didn't have time to check this today, I'll check the code tomorrow.